middlewares/validator.js

```diff
@@ -17,7 +17,7 @@ export default class Validator {
   static validateRegistration() {
     return [
       ...Validator.validateEmail(),
```

Comment:
#### Nitpick
- How will this look like? Can I see an example of this?

Reply:
<img width="478" alt="Screenshot 2019-03-14 at 4 19 06 AM" src="https://user-images.githubusercontent.com/8660204/54329061-7ce03400-4610-11e9-9680-d91aa429cf30.png">

Reply:
This is used with express-validator, which requires a validation error catching middleware and an array of checks.

Reply:
@valenstical , I meant how will the return array look like.